package/MDAnalysis/analysis/rdf.py

```diff
@@ -261,6 +273,13 @@ def _single_frame(self):
             mask = np.where(idxA != idxB)[0]
             dist = dist[mask]
 
+        if self.exclude_same:
+            # Ignore distances between atoms in the same residue
+            resixA = getattr(self.g1, self.exclude_same)[pairs[:, 0]]
```

Comment:
Nitpick but 

```suggestion 
            attr_ix_A = getattr(self.g1, self.exclude_same)[pairs[:, 0]]
```

and same with below as these are no longer resix? I could be  wrong?